### PR TITLE
updated search functionailty

### PR DESCRIPTION
**Pull request by Isabirye1515**
# Updated search functionality
[Screencast from 24-07-24 16:37:45.webm](https://github.com/user-attachments/assets/c19541d0-cb26-4617-b723-5f69d415ec79)
# THE CSS
The css i have was unavoisdable css thats all about position of the carbon react

### CODE REVIEW

**Comment by Isabirye1515** [comment]
please kindly review
@mozzy11 , @adityadeshlahre , @EdisonMeks , @caseyi , @rahul6603  , @Agaba-derrick , @manishjha-04

File: frontend/src/components/layout/Header.js
Comment (Isabirye1515) [inline frontend/src/components/layout/Header.js:1]: I own this commit @Isabirye1515

File: frontend/src/components/nonconform/common/ReportNonConformingEvent.jsx [hunk 1]
```diff
@@ -67,25 +67,23 @@ export const ReportNonConformingEvent = () => {
 
   const [nceForm, setnceForm] = useState({
     data: {
```
Comment (mozzy11) [inline frontend/src/components/nonconform/common/ReportNonConformingEvent.jsx:69]: Are these changes part of the issue youre working on ?

File: frontend/cypress/pages/NonConformPage.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ class NonConform {
     return cy.get("h2");
   }
 
-  getViewNonConformTitle(){
+  getViewNonConformTitle() {
```
Comment (mozzy11) [inline frontend/cypress/pages/NonConformPage.js:6]: Are these changes part of the issue youre working on ??

Comment (Isabirye1515) [reply frontend/cypress/pages/NonConformPage.js:6]: No sir,

Comment (Isabirye1515) [reply frontend/cypress/pages/NonConformPage.js:6]: How else can I resolve this

**Comment by manishjha-04** [comment]
Hey @Isabirye1515 ,

Everything looks good to me! Just a couple of suggestions:

- > Try to avoid using custom CSS if possible. You can implement all these styles using Grid, Row, and other functionalities from Carbon React.
- > The search bar is currently below the header nav, which looks a bit odd. Could you move it to the title bar instead? But please discuss this with @mozzy11  once before making the change.

Thanks!

File: frontend/cypress/pages/NonConformPage.js (hunk 2 above)
Comment (Isabirye1515) [reply frontend/cypress/pages/NonConformPage.js:6]: MR @mozzy11 if i happen to ask for , for narration on the way to remove the unwanted change i got it a little

**Review by mozzy11 — CHANGES_REQUESTED** [review]
@Isabirye1515 you have alot of unwabted changes. 
I see you also deleted soem files

**Pull request closed without merging**